app/src/main/res/layout/dialog_send_message.xml

```diff
@@ -1,15 +1,26 @@
-<?xml version="1.0" encoding="utf-8"?>
-<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
-    android:layout_width="match_parent"
-    android:layout_height="match_parent"
-    android:orientation="vertical">
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto">
 
-    <EditText
-        android:id="@+id/dialog_send_geo_message_edit_text"
-        android:layout_width="fill_parent"
-        android:layout_height="wrap_content"
-        android:paddingTop="20dp"
-        android:paddingLeft="20dp"
-        android:hint="Enter your message here..."
-        android:inputType="text" />
-</LinearLayout>
\ No newline at end of file
+    <data>
```

Comment:
Nevermind, we talked about our architecture, and the binding data will be the view model.